src/staking/staking_pool.h

```diff
@@ -0,0 +1,22 @@
+#ifndef ELECTRIC_CASH_STAKING_POOL_H
+#define ELECTRIC_CASH_STAKING_POOL_H
+#include "amount.h"
+
+class StakingPool {
```

Comment:
Naming convention of this codebase is to prefix every class name with a `C`, so this class should be called `CStakingPool`.

Reply:
done